### PR TITLE
long ass title something something freedom heart reagent

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
this pr adds a new reagent which currently its only use is to make a heart of freedom. you pour it on a normal heart to get it.

currently the only way to obtain this chemical is to grind a eagle hardsuit and helmet
#### Changelog

:cl:  
rscadd: Added a new chemical and a way to obtain the heart of freedom 
tweak: tweaked some text in heart of freedoms messages    
/:cl:

### CODE REVIEW

**Comment by CMOisLing** [comment]
Could you not put the description in the title?

**Comment by boodaliboo** [comment]
The Pr is done except for the chemical nans

**Comment by boodaliboo** [comment]
the only part of this thats wip is the chems name

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1499,6 +1499,19 @@
...
 		new /obj/item/clothing/shoes/galoshes/dry(t_loc)
+		
+/datum/reagent/americajuice
+	name = "America Juice"
+	description = "Despite its name it is not a juice"
+	reagent_state = LIQUID
+	color = "#A70FFF"
+	taste_description = "freedom"
+
+/datum/reagent/drying_agent/reaction_obj(obj/O, reac_volume)
```
Comment (SomeguyManperson) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Bruh

**Comment by SomeguyManperson** [comment]
Consult with the totally important lore team on whether America juice is a lore friendly name

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: whoops

**Comment by boodaliboo** [comment]
Why does this have stale I updated it yesterday

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]
See if you can find a better name but if not this is good enough for now, suggest asking about in the wiki/mentor channel or OOC for ideas. Also needs a full stop after the description!!

**Comment by boodaliboo** [comment]
ok will do

**Comment by boodaliboo** [comment]
im not sure whats needing to be done as when ive asked people said my name for it was ok

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1499,6 +1499,19 @@
 		var/t_loc = get_turf(O)
 		qdel(O)
 		new /obj/item/clothing/shoes/galoshes/dry(t_loc)
+		
+/datum/reagent/americajuice
+	name = "America Juice"
```
Comment (Hopekz) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	name = "Freedom Juice"
```
Here is a "suggestion" on a new name.

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
YHES

**Comment by alexkar598** [comment]
Going to close this because anyone grinding up their suit is a fucking moron and they probably wont exists

**Pull request closed without merging**